### DISCUSSION TITLE
Is wallet connect flutter SDK supports bitcoin transaction creation and transactions history?

### DISCUSSION THREAD

**Discussion by lironSendCrypto** [WalletConnect/WalletConnectFlutterV2#309] [Q&A]
Hello everyone

Is walletConnect flutter SDK supports bitcoin transaction creation and transactions history?
If not, are there any plans for future integration?

Thank you!

**Comment by quetool** [comment]
Hello @lironSendCrypto! Thanks for your interest! Regarding bitcoin there are still a few things that we are working on, once it's ready we will publish examples of the implementation. That should happen soon enough!

**Comment by faaqi** [comment]
Hey @quetool 

1. Can we get any rough time estimation regarding BTC support?

**Comment by quetool** [comment]
Hello @faaqi @lironSendCrypto! Specs for Bitcoin (and other chains) [got merged just yesterday](https://github.com/WalletConnect/walletconnect-docs/pull/1649)! I will be adding examples ASAP but you can start supporting it! https://walletconnect-docs-git-fork-towolabs-main-walletconnect1.vercel.app/advanced/multichain/rpc-reference/bitcoin-rpc

Please remind that only WalletConnectFlutterV2 and WalletConnectModal supports non-EVM chains, Web3Modal is only EVM compatible for now.

**Comment by lironSendCrypto** [comment]
This is perfect, thank you very much @quetool !

**Reply by quetool** [reply]
If you think the Question was resolved please could you mark the answer?
